index.html

```diff
@@ -0,0 +1,206 @@
+                        <div class="doDivHead">
+                            <h5>Plug vid</h5>
+                        </div>
+                        <div>
+                            <p class="doivpara">Plug vid is a videography service offered to 
+                                our clients with events or product commercials to
+                                aired on tv
+                            </p>
+                        </div>
+                    </div>
+                    <div class="doDiv">
+                        <div class="doDivImg">
+                            <img src="img/software.png" alt="Online Commerce">
+                        </div>
+                        <div class="doDivHead">
+                            <h5>Plug soft</h5>
+                        </div>
+                        <div>
+                            <p class="doivpara">Plug soft is a tech service offered
+                                to clients who need websites, application for their
+                                bussiness.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="doInno">
+                    <p class="doInnoPara">Lets innovate and create</p>
+                </div>
+                <div class="doLink">
+                    <p> <a href="" class="doLink">Get more information about us</a></p>
+                </div>
+            </div>
+            <div class="teamBack" id="tea">
+
+            </div>
+        </main>
+    <section>
+        <footer>
+            <div class="partners" id="pat">
+                <div class="partnersDiv Div1">
+                    Partners
+                    <hr class="Div1Hr">
+                </div>
+                <div class="partFle">
+                    <div class="partOn">
+                        <div class="partnersDiv Div2">
+                            Yo art
+                        </div>
+                        <div class="partnersDiv Div4">
+                            KPL
+                        </div>
+                        <div class="partnersDiv Div5">
+                            Foster
+                        </div>
+                    </div>
+                    <div class="partTw">
+                        <div class="partnersDiv Div3">
+                            GreenNet
+                        </div>
+                        <div class="partnersDiv Div4">
+                            Juggle
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="copyright">
+                <div class="copyDiv">
+                    <h3><img class="plugicon" src="img/plug_icon.png" alt=""> MY PLUG</h3>
+                </div>
+                <div class="copyDiv">
+                    2021 My plug company
+                    all rights reserved
+                </div>
+            </div>
+        </footer>
```

Comment:
**Changes Suggested**
- [ ] The `footer children` seem to be in the center of the page. Let us use flexbox's `justify-content: space-between` with some padding in order to depict the given design. 
- [ ] The two children of the `footer` element should be HTML5 `article` element. to follow the HTML5 semsntics.

Reply:
Fixed